### ISSUE TITLE
Probably stupid question: REBAR?

### ISSUE THREAD

**Issue by virtvoid** [trexminer/T-Rex#651]
Hi, just a probably stupid question: Is rebar support for 3000 series cards a thing for compute-stuff like this?

Or maybe miners wouldn't benefit at all and so no need to bother? I don't know, never done anything in this regard, that's why I wanted to bring this idea up. It should be supported via CUDA, from what I can grasp from a brief look at the Nvidia documentation.
Thanks!

**Comment by grenelt** [comment]
You don't need to bother - as long as the DAG fits into VRAM.
1. Maybe the so called zombie mode benefits from it, but gpus that lack memory don't support rebar and gpus that support rebar don't lack memory.
2. Rebar speeds up accessing VRAM - via PCIe bus. But this is typically limited bei 1x risers. So if there would be any benefit, you also have to speed up bus connection - using different mainboards, risers...

**Comment by virtvoid** [comment]
You're right, makes sense. Or better to say, no sense for this use-case. :)
